svelte-kit-scss/src/lib/components/IssueSearch/IssueSearchListItem/IssueLabel.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
```

Comment:
I wonder if this label is different from the labels under the **about** on the repo page? I wonder if it should actually be a shared component?
(not a request for change, just pondering)

Reply:
That color logic is not shared across all labels?